### ISSUE TITLE
Feature: fingerprint unlock

### ISSUE THREAD

**Issue by laurentdinclaux** [WhisperSystems/Signal-Android#6032]
Please add fingerprint support for unlocking the app as a complementary option to the plain passphrase.

Setup finger print -> type current passphrase and confirm -> input fingerprint -> passphrase get stored encrypted

Open app -> scan finger (or fallback to plain passphrase) -> the app decrypts the passphrase -> app unlocked

See: https://github.com/mcnerthney/android-fingerprint-decrypt

**Comment by ferranti** [comment]
I support this idea :)

**Comment by nrizzio** [comment]
Please don't deliberately delete the issue template and open duplicate issues (duplicate of #6031).
You can discuss this feature on the [community forum](https://whispersystems.discoursehosting.net) or mailing list, but this issue tracker [is not intended for discussion.](https://github.com/WhisperSystems/Signal-Android/blob/master/CONTRIBUTING.md#not-a-discussion-forum)

**Comment by 2-4601** [comment]
What @nrizzio said. Closing as duplicate.

**Comment by laurentdinclaux** [comment]
@nrizzio Thanks for being so kind! I opened a new one because you arbitrary closed the other one without giving a chance to discuss the feature request. A feature that get implemented in about any secure Android app these days.

You said:
> Please reopen if there's some modern keychain thing I don't know about which would enable that, though.

But because normal users like me cannot reopen a closed issue, and because I though you were aware of that, I though you meant opening a new one, which is what I did.
